cat-theme/src/js/main.js

```diff
@@ -0,0 +1,37 @@
+const app = (() => {
+	let body;
+	let menu_icon;
+	let menu_header;
+	let parallax;
+	
+	const init = () => {
+		body = document.querySelector('body');
+		menu_icon = document.querySelector('.menu-icon');
+		menu_header = document.querySelector('.menu-title');
+		menuItems = document.querySelectorAll('.nav__list-item');
+		parallax = document.querySelector(".parallax");
+
+		applyListeners();
+	}
+	
+	const applyListeners = () => {
+		menu_icon.addEventListener('click', () => toggleClass(body, 'nav-active'));
+		menu_header.addEventListener('click', () => toggleClass(body, 'nav-active'));
+
+		window.addEventListener('scroll', () => {
+			let offset = window.pageYOffset;
+
+			parallax.style.backgroundPositionY = offset * .08 + 'rem';
+		})
```

Comment:
Should we debounce this scroll method so it does not fire on every scroll on the page, but rather when someone stops scrolling?